shell/core/types-provisioning.ts

```diff
@@ -106,6 +106,11 @@ export interface IClusterProvisioner {
    */
   disabled?: boolean;
 
+  /**
+   * Hide the cluster provider card
+   */
+  hidden?: boolean;
```

Comment:
Not quite sure how we do changes to extension api changes. @aalves08 + @jordojordo , should this change include a bump to the shell package version indicating that a new release will be required at some point... or do we have another notification process?

This doesn't block the PR or customer (right now), but will when they upgrade / use 2.9.6

Reply:
Asked in slack, but also creted https://github.com/rancher/dashboard/issues/12930 for docs

Reply:
It should include a bump in the shell package + the release of that shell version via a tagged release on our repository. There's an internal doc that I can refer to.